docs/en/ui/README.md

```diff
@@ -8,6 +8,41 @@ The left side menu lists all available supported stacks with default dashboards.
 
 Follow the `Official Dashboards` menu to explore all default dashboards on their ways to monitor different tech stacks.
 
+## Sidebar Menu and Marketplace
+
+All available feature menu items are only listed in the marketplace(since 9.6.0). They are only visible on the Sidebar Menu when there are relative services
+being observed by various supported observation agents, such as installed language agents, service mesh platform, OTEL integration.
+
+The menu items defined in `ui-initialized-templates/menu.yaml` are the universal marketplace for all default-supported integration.
+The menu definition supports one and two levels items. The leave menu item should have the `path` and `dashboardID` for navigation, the icon of the level one item is defined from UI side.
+
+```yaml
+menus:
+  - name: General Service
+    icon: general_service
+    menus:
+      - name: Services
+        path: /general
+        dashboardID: General-Root
+      - name: Visual Database
+        path: /database
+        dashboardID: Virtual-Database-Root
```

Comment:
According to the discussion between @Fine0830 and me. We need to change this.
`name`, `icon`, and `layer` are required, others are not.
This is the runtime menu context.
![image](https://github.com/apache/skywalking/assets/5441976/7314a559-7e5a-4d4d-b370-a51b835ed74a)

path should be generated by UI side directly by `lower case` the `name`, and use `_` replacing the blank between words.

Also, this means the query protocol should be fixed as well, no `path`, but need to add `layer`.
https://github.com/apache/skywalking-query-protocol/blob/1f80cd0af8bc0baf71bb68961a894c110c9d9b79/ui-configuration.graphqls#L32

Reply:
I think the `name` should not contain special characters, including spaces.

Reply:
> I think the `name` should not contain special characters, including spaces.

So, in the 'Control Plane' of 'Service Mesh', the menu should be 'ControlPlane' and 'ServiceMesh'? Am I right?

Reply:
Right.

Reply:
> I think the `name` should not contain special characters, including spaces.

Then how the name is showing? We need `Service Mesh` text to display, that is my meaning of the name.

Reply:
> When menu item is alerting or dashboard, the value of layer should be empty?

Alerting and all menus are not configurable are excluded from this. Those would be hard coded only from UI side.

Reply:
> > When menu item is alerting or dashboard, the value of layer should be empty?
> 
> 
> 
> Alerting and all menus are not configurable are excluded from this. Those would be hard coded only from UI side.

Got it. It should be including dashboard settings, alert and settings.

Reply:
> > I think the `name` should not contain special characters, including spaces.
> 
> Then how the name is showing? We need `Service Mesh` text to display, that is my meaning of the name.

In the graph, the `name` means ID.  `title` could display on the page.